cms/templates/widgets/problem-edit.html

```diff
@@ -18,145 +18,156 @@
                         <span class="problem-editor-icon dropdown">
                             <img class="icon" src="${static.url('images/cms-editor_dropdown.png')}" alt="${_("Insert a dropdown response")}">
                         </span>
+                        ${_("Dropdown")}
                     </button>
                 </li>
                 <li>
                     <button type="button" class="explanation-button" data-tooltip="${_("Explanation")}">
                         <span class="problem-editor-icon explanation">
                             <img class="icon" src="${static.url('images/cms-editor_explanation.png')}" alt="${_("Add an explanation for this question")}">
                         </span>
+                        ${_("Explanation")}
                     </button>
                 </li>
             </ul>
             <ul class="editor-tabs">
                 <li><button type="button" class="xml-tab advanced-toggle" data-tab="xml">${_("Advanced Editor")}</button></li>
-                <li><button type="button" class="cheatsheet-toggle" data-tooltip="${_("Toggle Cheatsheet")}">?</button></li>
             </ul>
         </div>
         <textarea class="markdown-box">${markdown | h}</textarea>
-        %endif
-        <textarea class="xml-box" rows="8" cols="40">${data | h}</textarea>
-    </div>
-</section>
-
-<script type="text/template" id="simple-editor-cheatsheet">
-    <article class="simple-editor-cheatsheet">
-        <div class="cheatsheet-wrapper">
-            <div class="row">
-                <h6>${_("Heading")}</h6>
-                <div class="col sample heading-1">
-                    <img class="icon" src="${static.url('images/cms-editor_heading.png')}" alt="${_("Insert a heading")}">
-                </div>
-                <div class="col">
+            <article class="simple-editor-cheatsheet shown">
+                <div class="cheatsheet-wrapper">
+                    <div class="row">
+                        <h5>${_("Markdown Help")}</h5>
+                    </div>
+                    <div class="row">
+                        <h6>${_("Heading")}</h6>
+                        <div class="col sample heading-1">
+                            <img class="icon" src="${static.url('images/cms-editor_heading.png')}" alt="${_("Insert a heading")}">
+                        </div>
+                        <div class="col">
 <pre><code>H3
 =====
-</pre>
-                </div>
-            </div>
-            <div class="row">
-                <h6>${_("Multiple Choice")}</h6>
-                <div class="col sample multiple-choice">
-                    <img class="icon" src="${static.url('images/cms-editor_radio.png')}" alt="${_("Add a multiple choice question")}">
-                </div>
-                <div class="col">
+</code></pre>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <h6>${_("Multiple Choice")}</h6>
```

Comment:
Change 5: I would move these h6 items to be after the img class below and then update the styles to achieve the same visual layout as the button names in the markdown editor header, something like this: 


![image](https://user-images.githubusercontent.com/2023680/65065626-c9565480-d950-11e9-90a6-b89095d74f55.png)


I believe the necessary CSS changes to complement the html change are included below: #browser-hacking 

.col {
    display: block;
    &.sample {
      width: 60px; **REMOVED**
      margin-right: 30px;
      .icon {
        height: ($baseline * 1.25); **CHANGED TO 25px from 30px**
      }
    }
  }

  .cheatsheet-wrapper {
    padding: 10%;
  }
  h6 {
    margin-bottom: 7px;
    font-size: 15px;
    font-weight: 700;
    display: inline-block; **NEW / ADDED**
vertical-align: top;  **NEW / ADDED**
  }